btcpay-setup.sh

```diff
@@ -333,7 +333,14 @@ if [[ "$NBITCOIN_NETWORK" != "mainnet" ]] && [[ "$NBITCOIN_NETWORK" != "testnet"
     echo "NBITCOIN_NETWORK should be equal to mainnet, testnet or regtest"
 fi
 
-
+if [[ $BTCPAY_CRYPTOS == *"doge"* ]]; then
```

Comment:
Please move this to a separate PR. I am unsure if there will be consensus around adding altcoin specific code in the main setup.

Reply:
OK - will do - we need to do something though because static RPC user and pass will get users hacked eventually.

Reply:
Not really, the dogecoin node is not exposed externally, it is firewalled inside the docker network. Many of the other services (lightning for example) have a hardcoded rpcauth in their yml file

Reply:
When DOGE is launched, Docker shows it is NOT using the "docker network"
```
0.0.0.0:22555
```
is exposed on the host as the rpcport!

Reply:
That's only inside the docker container. It is not the actual host machine